packages/jest-transform/src/types.ts

```diff
@@ -40,10 +40,10 @@ export type TransformResult = TransformTypes.TransformResult;
 
 export interface CallerTransformOptions {
   // names are copied from babel: https://babeljs.io/docs/en/options#caller
-  supportsDynamicImport: boolean;
-  supportsExportNamespaceFrom: boolean;
-  supportsStaticESM: boolean;
-  supportsTopLevelAwait: boolean;
+  supportsDynamicImport?: boolean;
+  supportsExportNamespaceFrom?: boolean;
+  supportsStaticESM?: boolean;
+  supportsTopLevelAwait?: boolean;
 }
```

Comment:
why are these optional now?

Reply:
They are optional in Babel, so thought that was just missed when copying. https://babeljs.io/docs/en/options#caller

I'll gladly put them back, if this diff was intentional 🙈

Reply:
ah, right. Yeah, optional to pass to babel, be we always pass them in Jest 🙂 (both to transformers, and to babel for that matter)